corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -0,0 +1,52 @@
+{% extends "hqwebapp/two_column.html" %}
+{% load compress %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% requirejs_main 'reports/v2/js/views/caseSearch' %}
+
+{% block page_sidebar_classes %}sidebar-offcanvas col-xs-12 col-sm-3 col-lg-2{% endblock %}
+
+{% block page_row_classes %}row-offcanvas row-offcanvas-left{% endblock %}
+
+{% block page_content_classes %}col-xs-12 col-sm-9 col-lg-10{% endblock %}
+
+{% block js %}{{ block.super }}
```

Comment:
This block should be removed because this is a requirejs page.

Reply:
fixed here https://github.com/dimagi/commcare-hq/pull/22063/commits/5a5ad414ecb79a1b39fcb02f9be2e06830e86c80